generators/app/index.js

```diff
@@ -35,57 +37,80 @@ module.exports = Generator.extend({
     return this.prompt(prompts).then(function (props) {
       options = _.assign(options, props);
     });
-
-    // disabling this for now as pattern lab starter v8 doesn't really have drupal 7 or drupal 8 differences
-    // this.composeWith('pattern-lab-starter:extras', {options: options}, {
-    //   local: path.resolve(__dirname, '../extras')
-    // });
-
   },
 
   configuring: function () {
 
   },
 
   default: function () {
-    var dest = options.themePath ? path.resolve(process.cwd(), options.themePath) : './';
+    const release = options['release'] || 'master'
+    const release_path = release + '.tar.gz';
+    const compressed = _.last(_.split(release_path, '/'));
+    const decompressed = _.replace('particle-' + release, '/', '-');
+    const download_url = 'https://github.com/phase2/particle/archive/' + release_path;
+    const dest = options.themePath ? path.resolve(process.cwd(), options.themePath) : './';
+    const themeName = 'patternlab';
```

Comment:
`particle`?

Reply:
I'm not trying to bring this to generator-particle just yet, simply continue to pull down the PLS 9 repo. I suppose you could have created a whole new repo... meh.